Dockerfile

```diff
@@ -1,36 +1,20 @@
-FROM ubuntu:16.04
+FROM quay.io/ncigdc/bio-py36:latest
```

Comment:
I pulled out the `trimmomatic` and `fqvendorfail` packages into their own respective docker images. 

I will also open a PR in the relevant workflow with updates to calling those  tools.

Reply:
Got it, makes sense